websites/D/Daily Mail/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Daily Mail",
+	"description": {
+		"en": "The official website for the Daily Mail publication"
```

Consider using the official description from the site.
```suggestion
		"en": "MailOnline - get the latest breaking news, celebrity photos, viral videos, science & tech news, and top stories from MailOnline and the Daily Mail newspaper."
```